matrix/rules.py

```diff
@@ -345,7 +368,7 @@ def exception_handler(self, context, loop=None, exc_ctx=None):
 
     async def __call__(self):
         btask = self.loop.create_task(self.bus.notify(False))
-        context = self.load_suite(self.config_file)
+        context = await self.load_suite()
```

Comment:
Presumably, we don't want the UI or keyboard handling to hang while it's talking to the charm store.  Apparently, the CS is hitting 10s timeouts just for metadata requests occasionally, so I'm a bit wary of it being blocking.